website/content/docs/upgrading/version-guides/0.11.0.mdx

```diff
@@ -0,0 +1,173 @@
+---
+layout: docs
+page_title: Version 0.11.0 - Upgrade Guide
+description: |-
+  This upgrade guide describes any considerations for upgrading to Waypoint 0.11.0.
+---
+
+# Upgrading to Waypoint 0.11.0 from 0.10.x
+
+Waypoint 0.10.x can be upgraded to Waypoint 0.11.0 using a [standard upgrade](/waypoint/docs/upgrading#standard-upgrade).
+There are some considerations depending on what platform your server is installed on.
+There are no breaking changes in the protocol or `waypoint.hcl` configuration file.
+Due to the changes in `runner` management, we cannot guarantee that all Waypoint
+components (CLI, entrypoint, and server) are forwards and backwards
+compatible. We encourage users to have all components on the same version.
+
+## Server Snapshot
+
+Waypoint servers at versions 0.10.4 and 0.10.5 have a [bug that prevents
+snapshots from being taken](https://github.com/hashicorp/waypoint/pull/4523),
+if there is a dynamic configuration source set using the `waypoint config
+source-set` command, like so.
+
+If your server is running at either of these versions and a config
+source is set, then you will not be able to take snapshots using the command
+`server snapshot`. By default the `server upgrade` command will attempt to take
+a snapshot of the database before upgrading and this will fail due to this bug.
+
+There are a few options to upgrade to v0.11.0. If you do not need a snapshot of
+the database before upgrading, then you may run `server upgrade` with the flag
+`-snapshot=false` to proceed with upgrading, like so.
+
+```shell
+$ waypoint server upgrade -snapshot=false
+```
+
+Another alternative is to delete all config sources from Waypoint using
+`config source-set` with the `-delete` flag, and then running `server upgrade`.
+After the server is upgraded, you can re-create the config sources using
+`config source-set`.
+
+```shell
+# Repeat the command below for each config source that is set; this example
+# deletes the globally-scoped Vault config source
+$ waypoint config source-set -delete -type=vault -scope=global
+````
+
+A third approach is to use platform-native persistence solutions to back up the
+file system where the Waypoint database is stored. This includes Docker, AWS
+ECS, Kubernetes, and HashiCorp Nomad. The steps to do this with each of these
+platforms are detailed below.
+
+### Docker
+
+The Waypoint server stores data into a single `data.db` file, in Docker it's
+normally stored in `/data/data.db`. In the server data directory, copy the
+snapshot data and name the file `waypoint-restore.db`.
+
+Now run `waypoint server upgrade -snapshot=false`.
```

Comment:
For consistency, can we do this here and for the following 4 commands?
```suggestion
Now run:
```shell
 $ waypoint server upgrade -snapshot=false
tic marks * 3
```